docs/source/exceptions.rst

```diff
@@ -3,3 +3,4 @@ Exceptions
 
 .. automodule:: vws.exceptions
    :members:
+   :show-inheritance:
```

Comment:
Thank you.
Please also use `:inherited-members:` so that a reader can know all the properties of each exception without following the inheritance tree.